### PR TITLE
148 add a css app

### PR DESCRIPTION
**Pull request by Dan-Quin**
Only thing to still do would be to make the font changing command ever return invalid, but I tried a couple solutions and will prop just come back to this as it is relatively minor.

### CODE REVIEW

**Review by kavyab2006 — APPROVED** [review]

File: frontend/src/components/desktop_apps/screens/TermialAppScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+export default function TerminalAppScreen() {
+  const dispatch = useDispatch();
+  const [allPreviousCommands, setPreviousCommands] = useState<ICommand[]>([]);
+  const styleData = useSelector((state: IGameState) => state.styleData);
+  const checkForCommands = () => {
+    const inputElement = document.getElementById("command") as HTMLInputElement;
+    const inputValue = inputElement.value;
+    const command: ICommand = { input: inputValue, response: generateResponse(inputValue) };
+    setPreviousCommands([...allPreviousCommands, command]);
+    console.log(allPreviousCommands);
```
Comment (jvgusch) [inline frontend/src/components/desktop_apps/screens/TermialAppScreen.tsx:20]: Remove debugging statement

File: frontend/src/components/desktop_apps/screens/TitleBar.tsx [hunk 2]
```diff
@@ -8,19 +10,36 @@ interface TitleBarProps {
 
 export default function TitleBar({ open, setOpen }: TitleBarProps) {
   useKeyDown(setOpen, ["Escape"]);
+  console.log("AAAAAAAAAAAAAAA");
```
Comment (jvgusch) [inline frontend/src/components/desktop_apps/screens/TitleBar.tsx:13]: Remove debugging statement.

File: frontend/src/slices/styleDataSlice.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  }
+};
+
+const styleDataSlice = createSlice({
+  name: "styleData",
+  initialState: initialState,
+  reducers: {
+    setStyle: (state, action: PayloadAction<{ effect: string; target: string; value: string }>) => {
+      state[action.payload.effect][action.payload.target] = action.payload.value;
+      // console.log(action.payload.property);
```
Comment (jvgusch) [inline frontend/src/slices/styleDataSlice.ts:57]: Commented code can be removed.

**Review by jvgusch — APPROVED** [review]

File: frontend/src/components/desktop_apps/screens/TitleBar.tsx (hunk 2 above)
Comment (Dan-Quin) [reply frontend/src/components/desktop_apps/screens/TitleBar.tsx:13]: It has been removed.

File: frontend/src/slices/styleDataSlice.ts (hunk 3 above)
Comment (Dan-Quin) [reply frontend/src/slices/styleDataSlice.ts:57]: It has been removed.

**Review by jvgusch — APPROVED** [review]

**Review by emadsu — APPROVED** [review]

**Pull request merged**